numpy/core/include/numpy/ndarraytypes.h

```diff
@@ -1008,6 +1008,10 @@ typedef void (NpyIter_GetMultiIndexFunc)(NpyIter *iter,
 #define NPY_ITER_DELAY_BUFALLOC             0x00000800
 /* When NPY_KEEPORDER is specified, disable reversing negative-stride axes */
 #define NPY_ITER_DONT_NEGATE_STRIDES        0x00001000
+/* If output operands overlap with other operands (based on heuristics that
+   has false positives but no false negatives), make temporary copies to
```

Comment:
Nitpick: We try to format with all rows starting with a * and the last row with only `*/` on it.